package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "publish",
+  "version": "0.1.0-dev",
```

Comment:
A brief follow up conversation in the fluid-work channel. https://botbot.me/freenode/fluid-work/2015-10-20/?msg=52328727&page=1